scripts/identify-web-features.js

```diff
@@ -0,0 +1,303 @@
+  str += printStandardPositions(feature);
+  str += printUseCounter(feature);
+  str += printSurveys(feature);
+  str += printPreviousInterops(feature);
+  str += printWPTLink(feature);
+  str += `* **More information:** See the [web-features explorer](https://web-platform-dx.github.io/web-features-explorer/features/${feature.id}/).\n\n`;
+
+  return str;
+}
+
+// Post a new comment with the given markdown content or update an existing comment if it already exists.
+async function postOrUpdateComment(issueNumber, markdown) {
+  // Retrieve existing comments to check if we already posted a comment.
+  const commentsResponse = await octokit.request(`GET /repos/${argv.repo}/issues/${issueNumber}/comments`, {
+    headers: {
+      "X-GitHub-Api-Version": GITHUB_API_VERSION
+    }
+  });
+  const existingComment = commentsResponse.data.find(comment => comment.body.includes(HIDDEN_COMMENT_IN_ISSUE));
+
+  if (existingComment) {
+    // The bot already posted a comment. Update it.
+    console.log(`Updating existing comment #${existingComment.id}...`);
+    await octokit.request(`PATCH /repos/${argv.repo}/issues/comments/${existingComment.id}`, {
+      body: markdown,
+      headers: {
+        "X-GitHub-Api-Version": GITHUB_API_VERSION
+      }
+    });
+  } else {
+    // Post a new comment.
+    console.log(`Posting a new comment...`);
+    await octokit.request(`POST /repos/${argv.repo}/issues/${issueNumber}/comments`, {
+      body: markdown,
+      headers: {
+        "X-GitHub-Api-Version": GITHUB_API_VERSION
+      }
+    });
+  }
+}
+
+// The main entry point to the script.
+async function main() {
+  const issue = await getReferencedIssue();
+
+  console.log(`Processing issue #${issue.number}: "${issue.title}"`);
+  const featureIds = findFeaturesInIssue(issue);
+  const features = await Promise.all(featureIds.map(id => getFeatureData(id)));
+
+  if (features.length === 0) {
+    console.log("Could not find any matching features the issue body.");
+    return;
+  }
+
+  console.log(`Found ${features.length} matching feature(s):`);
+  console.log(features.map(f => `- ${f.id}`).join("\n"));
+
+  let content = "_This comment was automatically generated based on the information you provided. Please don't edit it._\n\n";
+  content += `Below is additional information about the web feature${features.length > 1 ? "s" : ""} (from the [web-features project](https://github.com/web-platform-dx/web-features/)) which ${features.length > 1 ? "are" : "is"} referenced in your proposal.`;
+  content += `This information might help motivate your focus area proposal.\n\n`;
+
+  for (const feature of features) {
+    const featureContent = getMarkdownContentForFeature(feature);
+
+    if (features.length > 1) {
+      content += `<details>\n`;
+      content += `<summary>${escapeFeatureName(feature)}</summary>\n\n`;
+      content += featureContent;
+      content += `</details>\n\n`;
+    } else {
+      content += featureContent;
+    }
+  }
+
```

Comment:
We need to handle the case where no features were found, with a helpful message like
"No web-features found in the proposal. If your proposal doesn't correspond to a web feature, that is fine. Otherwise please add the string `web-feature: <feature name>` to the initial comment to link your proposal to the corresponding feature".